wiki/Skin/en.md

```diff
@@ -1,10 +1,24 @@
----
-stub: true
----
-
 # Skin
 
-*See also: [Skinning](/wiki/Skinning)*\
-*For the dedicated sub-forum for skins, see [Skinning](https://osu.ppy.sh/community/forums/15)*
+A **skin** is a set of graphics (images) and custom sample sets (audio clips) that replace the default ones used by osu!. Many players use skins to allow for a more customised user experience. Most of the important UI elements and sounds in osu! can be "skinned". 
+
+## Reasons to Use Skins
+
+- Skins can be used to further optimize a player's gameplay, since many skins hide certain visual details or offer gameplay elements focused on a certain playstyle like specific mod combinations.
+- In addition to gameplay focused changes, skins can also offer custom menus which commonly feature characters from various anime and games, as well as being based around certain design styles, like minimalism or techno.
+
+## How to Use Custom Skins
+
+Skins can either be [created from scratch](/wiki/Skinning), or downloaded as a `.osk` file and imported into the game. To import an `.osk` file simply open it with osu!, or drag-and-drop it into osu!. If a skin you download is in `.zip` or `.rar` format unzip it and move it into the `Skins` folder of your osu! installation. Make sure that you are moving the folder containing the `skin.ini` and not a folder that contains the folder.
+
+## Popular Skin Websites
+
+- [Work In Progress Skins forum](https://osu.ppy.sh/community/forums/119)
+- [Remixed Skins forum](https://osu.ppy.sh/community/forums/124)
+- [Completed Skins forum](https://osu.ppy.sh/community/forums/109)
+- [Completed Skins Compendium](https://compendium.skinship.xyz/)<!-- TODO: explain how it expands upon completed -->
+- [Circle People Pro Player Skin Collection](https://circle-people.com/skins/)
+
+Side Note: Most osu! players these days use custom skins, with many of the high-ranking ones either linking them on their userpage, in YouTube video descriptions or providing it via a Twitch chat command.
 
-A **skin** is a set of graphics (images) and custom sample sets (audio clips) that replace the default ones used by osu!. Many players regularly engage in the use of skins to allow for a more customised user experience, should they dislike the default skin. Almost all of what is visible or audible in osu! can be "skinned."
+- See also: [Skinning](/wiki/Skinning)
```

Comment:
I think this is irrelevant becuase there's already a link to the skinning article (alternatively, you could move this back to a note under the `# Skin` heading)
```suggestion
```